crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "sui-graphql-rpc"
+version.workspace = true
```

Comment:
Let's decouple the version from the beginning.

Reply:
Ah just saw this comment. Will fix in next PR